src/toolchain/export-android.md

```diff
@@ -0,0 +1,112 @@
+<!--
+ ~ Copyright (c) godot-rust; Bromeon and contributors.
+ ~ This Source Code Form is subject to the terms of the Mozilla Public
+ ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+ ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Export to Android
+
+```admonish note title="Android CLI and JDK 17"
+Keep track of where you install the Android Command Line Tools and JDK 17
+```
```

Comment:
What does "keep track" mean? Notes should ideally be pieces of knowledge, or actionable.

Reply:
Ya I guess this part really isn't needed.  Its common sense that you need to know where the software you want to run is installed.